Aliucord/src/main/java/com/aliucord/coreplugins/accountstanding/ViolationPage.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * This file is part of Aliucord, an Android Discord client mod.
+ * Copyright (c) 2025 Juby210 & Vendicated
+ * Licensed under the Open Software License version 3.0
+ */
+
+package com.aliucord.coreplugins.accountstanding
+
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import androidx.core.content.res.ResourcesCompat
+import com.aliucord.*
+import com.aliucord.fragments.SettingsPage
+import com.aliucord.utils.DimenUtils.dp
+import com.aliucord.utils.MDUtils
+import com.aliucord.utils.ViewUtils.addTo
+import com.lytefast.flexinput.R
+import com.aliucord.utils.ViewUtils.findViewById
+import com.discord.utilities.color.ColorCompat
+import com.discord.utilities.time.ClockFactory
+import com.discord.utilities.time.TimeUtils
+import com.discord.utilities.SnowflakeUtils
+import com.discord.api.utcdatetime.UtcDateTime
+
+class ViolationPage(val violation: String, val flaggedContent: String?, val actions: List<String>, var id: Long, val maxExpirationTime: UtcDateTime) : SettingsPage() {
```

Comment:
This should also be internal
For future work, anything in core that isn't API should be internal or private if possible